src/main/java/org/example/MostCommon.java

```diff
@@ -0,0 +1,49 @@
+package org.example;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.nio.charset.Charset;
+public class MostCommon {
+    private String readFile(String path, Charset encoding) throws IOException
+    {
```

Comment:
This formatting is a C#-style formatting. Are you using the formatter of your IDE of choice?